dynamicio/mixins/with_local.py

```diff
@@ -0,0 +1,274 @@
+# pylint: disable=no-member, protected-access, too-few-public-methods
+
+"""This module provides mixins that are providing Local FS I/O support."""
+
+import glob
+import os
+from contextlib import contextmanager
+from threading import Lock
+from typing import Any, Mapping, MutableMapping, Optional
+
+import pandas as pd  # type: ignore
+from fastparquet import ParquetFile, write  # type: ignore
+from pyarrow.parquet import read_table, write_table  # type: ignore # pylint: disable=no-name-in-module
+
+from . import utils
```

Comment:
I have no particular issue with using relative imports, but in general we avoid them as a convention unless we want to impose a specific structure in a repo/lib. Happy to leave it as is, but if you are not doing this for a specific reason then lets not use relative imports please.